data/report_templates/project_summary.md

```diff
@@ -17,6 +17,11 @@ NGI Project Name
 NGI Project ID
 :   {{ project.ngi_id }}
 
+{% if project.ngi_facility %}
+NGI Facility
+:   Genomics {{ project.ngi_facility }} Stockholm
```

Comment:
This should be entirely generated in the Stockholm specific code, as this template will be used in Uppsala too.